en/development/errors.rst

```diff
@@ -204,17 +205,21 @@ For more details on HTTP 4xx error status codes see :rfc:`2616#section-10.4`.
 
 For more details on HTTP 5xx error status codes see :rfc:`2616#section-10.5`.
 
+
 You can throw these exceptions from you controllers to indicate failure states,
```

Comment:
@ions Just because I sde it now, would you please update also ``from you**r** controller`` here ?